zkSNACKs/WalletWasabi
# [VDG] Coin Control - Stage 1

This is the simplified (non-dynamic) version of the Coin Control feature (https://github.com/zkSNACKs/WalletWasabi/pull/9350).

![image](https://user-images.githubusercontent.com/3109851/198555108-1fd631ed-0785-4edb-ba80-4ada2ff2bc26.png)

The strategy with this bigger feature is the following:

- Create the feature in several stages (different PRs).
- Review each PR to make it easier to review and understand.
- Implement a kind of [*feature toggle*](https://en.wikipedia.org/wiki/Feature_toggle) so only developers can test the feature while it evolves. 

This PR corresponds to the Stage 1 of the feature.

What's included?

- The coin list with the final appearance and the correct data. 

What is not included?

- Actual coin selection.
- Filtering.
- Selected amount, remaining amount...
- User cannot continue.

What **will not be** included?

- Dynamic updates of the coin list. The coin list and its properties are fetch when the dialog opens and will not be refreshed afterwards. This means that:
   - Edge cases like receiving coins while the dialog is open will not be covered.
   - Statuses (`confirmed`, `is coinjoining`, `banned until`...) will not vary at all. 

If users want to refresh the list, they will have to do a close/open cycle.

## How to test

1. Press the LEFT ALT key while in the Transaction Preview screen (summary)
2. A new hyperlink button will appear. 
3. Click on it.

Remember that you won't be able to do anything on it just yet.
